lib/matplotlib/backends/backend_qt.py

```diff
@@ -405,8 +405,7 @@ def start_event_loop(self, timeout=0):
             raise RuntimeError("Event loop already running")
         self._event_loop = event_loop = QtCore.QEventLoop()
         if timeout > 0:
-            timer = QtCore.QTimer.singleShot(int(timeout * 1000),
-                                             event_loop.quit)
```

Comment:
Did you check that it's OK to get the timer get gc'ed immediately?  e.g. what happens if you insert a gc.collect() right here (gc'ing the timer); does a e.g. 2s timer get run?
Perhaps it is OK, but it's never obvious with GUI toolkits...

Reply:
Not really. I'll replace it with `_`.